Update voice-and-tone.mdx (#2850)
grommet/hpe-design-system: aries-site/src/pages/foundation/voice-and-tone.mdx

| Strong all the more              | _makes the case strong all the more._                                                  | _makes the case stronger._                                                                                                                                            | Never used.                                                    |
| I have a doubt                   | _I have a doubt about that._                                                           | _I have a question_                                                                                                                                                   | "I have a doubt about that" implies you think it is incorrect. |

> ## Point of view (POV)It is equated with yelling and increases stress levels.

Reviewer: Why is this sentence here?

## Capitalization

> There are four types of capitalization. 

Reviewer 2: There are more than four types.  There is "all caps", but we don't use it.  Also, camelCase usually refers to words where the first letter is NOT capitalized (hence, the camel reference).  PascalCase is the common term for when the first lettrer of each word is capitalized.

Author: Yes, I forgot all caps. I wrote this a little quickly. In the original version, which I did find, I had 5 lol

Author: I've edited this.

> 1. Sentence case, where the first letter is a capital, the second and all subsequent words begin with lowercase letters.

Reviewer: ```suggestion
1. Sentence case, where the first word is capitalized and all subsequent words begin with lowercase letters.
```

> 1. Sentence case, where the first letter is a capital, the second and all subsequent words begin with lowercase letters.

Reviewer 2: all subsequent words are not always lowercase in a sentence-case sentence.  Proper nouns should always be capitalized.  BTW, if we don't already, we should talk about not inventing new proper nouns.  For example, folks tend to want to turn any tech term into a proper noun: Send an Email regarding the Power Button in order to Reset the System. (none of these are proper nouns)

Author: Re: "subsequent" you're right! I just had to rewrite this from memory quickly because I thought my branch was lost and I want to be able to have this published by Tuesday for the presentation to GLC. I will re-word :) TY!

Also, I forwarded you something that Edgar had written. It's a table that also has a column for how big of an impact it would make on their systems. If your schedule allows @ericsoderberghp could we talk about this soon?

Author: The text that's already published states "Fight the urge to elevate normal nouns to proper noun status..." ;)

   (example: Storage volume) 
2. Title case, where most words are capitalized except for "connector" words such as "the," "an," "or," etc.
   (example: "The Benefits of Cloud Storage Explained")
3. Camel case, where there are capital letters inside the word, almost always for branding purposes.
    (example: GreenLake)
4. All lower case, where every single word is lowercase 
    (example: "lower case exudes an approachable and casual vibe")

At HPE, almost all text should be written in sentence case. 

> Sentence case:

Reviewer: ```suggestion
Sentence case should be used for:
```
and similar pattern for the rest

Author: Does this sound good?

"Sentence case should be applied on:"

> Sentence case:

Reviewer 2: I emailed a larger list of things to consider.  It would be good to be exhaustive here.

Author: TY, Edgar, yes, will absolutely look that over -maybe with Eric- and add what we're missing.

- Attribute labels
- Column headings
- Legends
- Tabs
- Section and subsection headings
- Paragraphs of text
- Tips
- Notifications
- Help

Title case:
- Page/dialog titles
- Buttons

Capitalize proper nouns. HPE brand and product names are proper nouns.

Fight the urge to elevate normal nouns to proper noun status. For example, if you are in a "data sources" page and need to say something about data sources, do not capitalize it. Unless it is a true proper noun, leave it uncapitalized when following sentence capitalization rules.

> DO NOT USE ALL CAPS IN EITHER TITLES OR SENTENCES. It is equated with yelling and increases stress levels.

Reviewer: Wondering if we shouldn't use all caps here

Author: @ericsoderberghp ? opinion?

Reviewer 3: Agreed with Taylor here. I think we should always model the correct behavior unless clearly in a "Do not" section.

## Capitalization

There are five types of capitalization. 
1. Sentence case, where the first letter is capitalized and subsequent words begin with lowercase letters, with some exceptions, such as proper nouns.
   (example: Storage volume) 
2. Title case, where most words are capitalized except for "connector" words such as "the," "an," "or," etc.
   (example: "The Benefits of Cloud Storage Explained")
3. PascalCase, where there are capital letters inside the word, almost always for branding purposes.
    (example: GreenLake)
4. All lower case, where every single word is lower case 
    (example: "lower case exudes an approachable and casual vibe")
5. All caps, where every letter is capitalized
    (example: ALL LETTERS ARE CAPITALIZED)

> At HPE, almost all text should be written in sentence case. Never begin a sentence or phrase with a lower case letter because it is too casual for our brand. Do not write in all caps because it is equated with yelling and increases stress.

Reviewer: ```suggestion
At HPE, almost all text should be written in sentence case. Do not write in all caps because it is equated with yelling and increases stress.
```
If you start with a lowercase letter, it's not sentence case -- so can we remove this redundant sentence?

Sentence case should be applied on:
- Attribute labels
- Column headings
- Legends
- Tabs
- Menus
- Section and subsection headings
- Paragraphs of text
- Tips
- Notifications
- Help

> - Buttons

Reviewer: I believe previously we've been told that Button labels like "Sign Up" etc. should use title case -- has that guidance been modified? If so, we'll need to modify our site examples and a dev ticket should be created.

Reviewer 2: Button case should match the case of the button target.  And, because page titles and dialog titles should be sentence case, the buttons should also be sentence case.

Reviewer: I was able to find the [initial ticket and discussion](https://github.com/grommet/hpe-design-system/issues/937#issuecomment-651214534) where we updated certain button labels like "Sign Up" to use title case.

Based on this screen grab from Brand Central's Writing Style Guide, it seems that there are some use cases where it is appropriate to title case a button label.

<img width="903" alt="Screen Shot 2022-08-26 at 1 06 15 PM" src="https://user-images.githubusercontent.com/12522275/186983309-eba82f7a-e614-45d2-a6e9-76543b2cadcf.png">

I agree with you Edgar that button case should match the case of the button target. Do you think it would be helpful to highlight some of the cases that aren't sentence case so that there's more clarity?

Reviewer 2: If there is a general rule with limited specific exceptions, we should list the exceptions.

Reviewer: @dahliaferrer we should highlight these limited specific exceptions.

- Dialog titles

Title case should only be applied on:
- Page titles

> - Subsections/subheadings

Reviewer: This seems counter to the direction we are moving on the site. We have been adjusting all subsection headings to use sentence case.

Reviewer 2: Agree.  I thought we were changing everything, last time we spoke.

Reviewer: @dahliaferrer noticing this content has not been moved. Shouldn't "subsections/subheadings" be under the sentence case section?

Reviewer: Edit, I see you added it above but didn't remove it from here.

Capitalize proper nouns. HPE brand and product names are proper nouns.

Fight the urge to elevate normal nouns to proper noun status. For example, if you are in a "data sources" page and need to say something about data sources, do not capitalize it. Unless it is a true proper noun, leave it uncapitalized when following sentence capitalization rules.

When writing instructions, you can choose to refer generally to the action that should be taken or to the button or page. When you refer generally to an action, use lower case. When you refer to a specific button, page or other element, follow the case of the element.  

Example of when you can refer to action in a general manner:

> To create a data fabric you must first [anchor]create a new cluster[anchor].

Reviewer: ```suggestion
To create a data fabric you must first create a new cluster.
```

Reviewer 2: The whole point of these examples is to show when anchors should follow capitalization based on what they refer to vs following sentence capitalization rules.  We should not make this change.

vs.

Example when you refer to a page or button:

> First, go to the "[anchor]Create a New Cluster[anchor]" page to create a data fabric.

Reviewer 2: Don't make this change either.  It is a correct example and becomes incorrect with the change.

> First, go to the "[anchor]Create a New Cluster[anchor]" page to create a data fabric.

Reviewer: I see, I did not follow what the markup was intending

> ## Acronyms and Initialisms

Reviewer: ```suggestion
## Acronyms and initialisms
```
sentence case?

  - Incorrect: _One of the input parameters is null._
  - Correct: _One of the parameters is null._ or _One of the specified parameters is null._ or _The "name" parameter is null._

> ## Other Common Problems and How to Address Them

Reviewer: ```suggestion
## Other common problems and how to address them
```
sentence case?
